vividus/src/test/java/org/vividus/configuration/EncryptedPropertiesProcessorTests.java

```diff
@@ -50,7 +50,8 @@ class EncryptedPropertiesProcessorTests
     @CsvSource({
             ENCRYPTED_VALUE_PLACEHOLDER + ", " + DECRYPTED_VALUE,
             "username=open password=" + ENCRYPTED_VALUE_PLACEHOLDER + " secret-key=" + ENCRYPTED_VALUE_PLACEHOLDER
-                    + ", username=open password=" + DECRYPTED_VALUE + " secret-key=" + DECRYPTED_VALUE
+                    + ", username=open password=" + ENCRYPTED_VALUE_PLACEHOLDER
+                    + " secret-key=" + ENCRYPTED_VALUE_PLACEHOLDER
```

Comment:
that's a wrong change, properties are not decrypted anymore

Reply:
if it make on Vault processor level ?